README.md

```diff
@@ -71,7 +71,7 @@ docker-compose build test
 After building the images/containers, the tests can be ran with:
 
 ```bash
-docker-compose up test mongo --exit-code-from test
+docker-compose up --exit-code-from test test mongo
```

Comment:
`test` service actually has a depends_on on mongo, so you only need to start the `test` service 
```suggestion
docker-compose up --exit-code-from test test
```